example/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 
 services:
   mongodb:
-    image: mongo:3.2
+    image: mongo:4.0
```

Comment:
should we split upgrades and adding PM2 in 2 commits?

Reply:
Yes we can

Reply:
Done